src/interfaces/type-annotations-and-interfaces.ts

```diff
@@ -0,0 +1,32 @@
+// Interfaces and improving code readability
+
+// Long type annotation without interfaces
```

Comment:
we could be much more concise on this lesson by writing something like: 

`//Long type annotation is shown as inline declaration of type `

`..codeBlock
`

`//Reusable interface which improves code readability (preferred way) `

`..codeBlock`